### DISCUSSION TITLE
GUI

### DISCUSSION THREAD

**Discussion by fm-sys** [Krrishdhaneja/virtual-assistants#18] [General]
https://user-images.githubusercontent.com/64581222/103454990-7fca7e00-4ce9-11eb-8084-2456623468e6.mp4

Here is a first GUI scratch. The GUI is created from a mixture of PyQt5, HTML, CSS and JS... (Of course, however, it is still far from being able to be used)

**Comment by fm-sys** [comment]
@Krrishdhaneja Do you want to create an icon for Friday? 

PNG
256x256px 
with transparent background

**Reply by Krrishdhaneja** [reply]
> @Krrishdhaneja Do you want to create an icon for Friday?
> 
> PNG
> 256x256px
> with transparent background

Yes, I want to but what the logo would be, i Also was having a thought of creating an icon but don’t know what the logo could be ?

**Reply by fm-sys** [reply]
Be creative! ;-) 

You could also create a 1080x1080 version which we could show in the main UI  before the first speech bubble

**Reply by Krrishdhaneja** [reply]
Um.. the text part is working great but will it be as responsive as text in speech synthesis? Cuz when i tried to make this in tkinter the gui would usually appear when the program is ended cuz tk.mainloop() is at end and if i would be putting it in front of any other function of Friday that wouldn’t work.

**Reply by Krrishdhaneja** [reply]
Anything resolved in snapdrop issue of RobinLinus?

**Reply by fm-sys** [reply]
> Um.. the text part is working great but will it be as responsive as text in speech synthesis? Cuz when i tried to make this in tkinter the gui would usually appear when the program is ended cuz tk.mainloop() is at end and if i would be putting it in front of any other function of Friday that wouldn’t work.

Yep. You cannot simply use mainloop() or something like that. But I think I will manage it, nothing is impossible...

**Reply by Krrishdhaneja** [reply]
Great positivity!👍

**Comment by Krrishdhaneja** [comment]
> Capture_20210102105511_Trim.mp4 
> 
> Here is a first GUI scratch. The GUI is created from a mixture of PyQt5, HTML, CSS and JS... (Of course, however, it is still far from being able to be used)

That is Awesome! The GUI I created was not as good as yours!😅.
